docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -35,7 +35,10 @@ Excel add-ins run across multiple versions of Office, including Office 2016 or l
 | [ExcelApi 1.1](excel-api-1-1-requirement-set.md)  | Version 1509 (Build 4266.1001) or later   | 1.19 or later | 15.20 or later | January 2016 |
 
 > [!NOTE]
-> The build number for Office 2016 installed via MSI is 16.0.4266.1001. This version only contains the ExcelApi 1.1 requirement set.
+> Users on perpetual versions of Office will be restricted to certain requirement set.
+>
+> - Office 2019 contains ExcelApi 1.8 and earlier.
+> - Office 2016 only contains the ExcelApi 1.1 requirement set.
```

Comment:
If you make the change I suggested earlier, consider rephrasing this to -> 

Office 2016 supports only the ExcelApi 1.1 requirement set.